url.bs

```diff
@@ -1068,6 +1068,33 @@ code point is U+003A (:).
 <p class="note">As per the <a href=#url-writing>URL writing</a> section, only a
 <a>normalized Windows drive letter</a> is conforming.
 
+<p>A string <dfn>starts with Windows drive letter</dfn> if all of the following are true:
+
+<ul>
```

Comment:
Probably better to use `<ul class=brief>` here without `<p>`s to make it more of a single paragraph.